web/settings.py

```
@@ -363,3 +363,43 @@
+    
+    # For Docker/container environments, dynamically add the host IP
+    import socket
+    hostname, _, ips = socket.gethostbyname_ex(socket.gethostname())
+    INTERNAL_IPS += [ip[:-1] + "1" for ip in ips]
+TEST_RUNNER = "django.test.runner.DiscoverRunner"
+
+if "test" in sys.argv:
+    CACHES = {
+        "default": {
+            "BACKEND": "django.core.cache.backends.dummy.DummyCache",
+        }
```

Ruff (0.8.2)

Trailing comma missing

Add trailing comma

(COM812)